code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -143,6 +143,39 @@
 		reagents.add_reagent("ammonia", 60)
 		update_icon()
 
+/obj/item/weapon/reagent_containers/glass/bottle/eznutrient
+	name = "\improper EZ NUtrient bottle"
+	desc = "A small bottle."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle-4"
+
+	New()

```

Comment:
Adjusting all reagent containers to current code standards is not in scope of this PR.